.github/workflows/scripts/validate-coderabbit.yml

```diff
@@ -11,7 +11,6 @@ jobs:
     CodeRabbit-Approval:
         name: Validate CodeRabbit.ai Approval
         runs-on: ubuntu-latest
-        needs : [Code-Quality-Checks, Test-Application, Start-App-Without-Docker, Docker-Start-Check]
```

Comment:
1. The YAML file mustn't be in the scripts directory
2. Create a script with all the run statements in it.
3. Move the YAML back to the pull-requests.yml file

Reply:
oh ok let me try to fix it